spec/core_functions/meta/get_mixin/error.hrx

```diff
@@ -0,0 +1,262 @@
+
+<===>
+================================================================================
+<===> argument/too_few/input.scss
+@use "sass:meta";
+a {b: meta.get-mixin()}
+
+<===> argument/too_few/error
+Error: Missing argument $name.
+  ,--> input.scss
+2 | a {b: meta.get-mixin()}
+  |       ^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:meta
+1 | @function get-mixin($name, $module: null) {
+  |           =============================== declaration
+  '
+  input.scss 2:7  root stylesheet
+
+<===>
+================================================================================
+<===> argument/too_many/input.scss
+@use "sass:meta";
+a {b: meta.get-mixin(c, true, d, e)}
+
+<===> argument/too_many/error
+Error: Only 2 arguments allowed, but 4 were passed.
+  ,--> input.scss
+2 | a {b: meta.get-mixin(c, true, d, e)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:meta
+1 | @function get-mixin($name, $module: null) {
+  |           =============================== declaration
+  '
+  input.scss 2:7  root stylesheet
+
+<===>
+================================================================================
+<===> non_existent/input.scss
+@use "sass:meta";
+a {b: meta.get-mixin(does-not-exist)}
+
+<===> non_existent/error
+Error: Mixin not found: does-not-exist
+  ,
+2 | a {b: meta.get-mixin(does-not-exist)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 2:7  root stylesheet
+
+<===>
+================================================================================
+<===> through_forward/show/input.scss
+@use "sass:meta";
+@use "midstream" as *;
+a {
+  @include meta.apply(meta.get-mixin(d));
+}
+
+<===> through_forward/show/_midstream.scss
+@forward "upstream" show c;
+
+<===> through_forward/show/_upstream.scss
+@mixin d() {a: c}
+
+<===> through_forward/show/error
+Error: Mixin not found: d
+  ,
+4 |   @include meta.apply(meta.get-mixin(d));
+  |                       ^^^^^^^^^^^^^^^^^
+  '
+  input.scss 4:23  root stylesheet
+
+<===>
+================================================================================
+<===> through_forward/hide/input.scss
+@use "sass:meta";
+@use "midstream" as *;
+a {
+  @include meta.apply(meta.get-mixin(c));
+}
+
+<===> through_forward/hide/_midstream.scss
+@forward "upstream" hide c;
+
+<===> through_forward/hide/_upstream.scss
+@mixin c() {a: c}
+
+<===> through_forward/hide/error
+Error: Mixin not found: c
+  ,
+4 |   @include meta.apply(meta.get-mixin(c));
+  |                       ^^^^^^^^^^^^^^^^^
+  '
+  input.scss 4:23  root stylesheet
+
+<===>
+================================================================================
+<===> division/input.scss
+@use "sass:meta";
+@mixin foo() {}
+@mixin bar() {}
+a {b: meta.get-mixin(foo) / meta.get-mixin(bar)}
+
+<===> division/error
+Error: get-mixin("foo") isn't a valid CSS value.
+  ,
+4 | a {b: meta.get-mixin(foo) / meta.get-mixin(bar)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 4:7  root stylesheet
```

Comment:
This should probably be in `spec/values/mixins` since it's really testing the behavior of the mixin type rather than the `meta.get-mixin` function. Also, test the other operations.